public/locales/en/settings.json

```
@@ -27,5 +27,21 @@
   "autoAddScreenshots": "Auto add screenshots",
   "autoAddScreenshotsDescription": "Use <0>{ctrlKey}</0> + <2>{altKey}</2> + <3>S</3> to take screenshots and add them to the repository.",
   "downloadCopiedHash": "Download copied hash",
-  "downloadCopiedHashDescription": "Use <0>{ctrlKey}</0> + <2>{altKey}</2> + <3>D</3> to download the last copied hash."
+  "downloadCopiedHashDescription": "Use <0>{ctrlKey}</0> + <2>{altKey}</2> + <3>D</3> to download the last copied hash.",
+  "AnalyticsToggle": {
+    "label": "Help improve this app by sending anonymous usage data",
+    "summary": "What data is collected?",
+    "paragraph1": "IPFS hosts a <1>Countly</1> instance to record anonymous usage data for this app.",
```

May be just me, but feels we should clarify we mean Shipyard||Community||Project, not the protocol itself.

```suggestion
    "paragraph1": "IPFS Shipyard hosts a <1>Countly</1> instance to record anonymous usage data for this app.",
```